nextflow_schema.json

```diff
@@ -163,6 +164,15 @@
                     "fa_icon": "fas fa-question-circle",
                     "hidden": true
                 },
+                "publish_dir_mode": {
```

Comment:
Something to update in `modules.conf` already?

Reply:
Updated it for all modules